python/pythonmonkey/lib/pmjs/global-init.js

```diff
@@ -17,6 +17,37 @@
 for (let mid in require.cache)
   delete require.cache[mid];
 
+/* Recreate the python object as an EventEmitter */
+const { EventEmitter } = require('events');
+const originalPython = globalThis.python;
+const python = globalThis.python = new EventEmitter('python');
+Object.assign(python, originalPython);
+
+/* Emulate node's process.on('error') behaviour with python.on('error'). */
+python.on('error', function unhandledError(error) 
+{
+  if (python.listenerCount('error') > 1)
+    return;
+  if (python.listenerCount('error') === 0 || python.listeners('error')[0] === unhandledError)
+    python.emit('unhandledException', error);
+});
+
```

Comment:
Please see
https://github.com/Distributive-Network/PythonMonkey/commit/b30b57069d48ff3ef7b2dcc2bbea5bc2dbc59a90

Reply:
The code in this PR is very old.